package.json

```diff
@@ -64,11 +63,16 @@
     "eslint": "3.6.0",
     "font-awesome": "4.6.3",
     "handlebars": "4.0.5",
+    "imports-loader": "0.6.5",
     "istanbul": "0.4.5",
+    "jquery": "2.1.1",
+    "jquery-ui": "1.12.1",
```

Comment:
Version in repo is 1.11.1. Shouldn't we merge this PR with same versions and improve/test later on? That seems safer to me (like you did with jQuery, staying on 2.1.1 even though latest v2 is 2.2.4 and there is even a v3).